### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**
## Brief description of the changes

Fixed some typos
## What browsers and operating systems have you tested these changes on?

N/A
## Are all automated tests passing?

yes
## Is this pull request against develop or some other non-master branch?

yes

### CODE REVIEW

File: docs/api/methods.jmd [hunk 1]
```diff
@@ -327,13 +327,13 @@ A `CanvasWrapper` object:
     {
         "name": "id",
         "type": "Integer",
-        "description": "The fild id."
+        "description": "The field id."
```
Comment (rnicholus) [inline docs/api/methods.jmd]: This one should be "file" instead of "field".

**Comment by rnicholus** [comment]
Wow, that's a lot of typos. Thanks for doing this! Can you update your branch based on my suggestion? After that, I'll merge this into develop.

**Comment by pborreli** [comment]
Done

**Comment by rnicholus** [comment]
Great, thanks for your help!

**Pull request merged**